polaris/polaris/sep38/manager_api.py

```diff
@@ -0,0 +1,232 @@
+from django.forms import model_to_dict
```

Comment:
I know this wasn't a trivial piece of code to write but there isn't a need for it, and allows external clients to change the anchor's data model, which is a security concern. Lets remove this file.